http/response/builder.go

```diff
@@ -96,7 +96,7 @@ func (b *Builder) writeHeaders() {
 	b.headers["X-XSS-Protection"] = "1; mode=block"
 	b.headers["X-Content-Type-Options"] = "nosniff"
 	b.headers["X-Frame-Options"] = "DENY"
-	b.headers["Content-Security-Policy"] = "default-src 'self'; img-src *; media-src *; frame-src *; child-src *"
+	b.headers["Content-Security-Policy"] = "default-src 'self'; img-src *; media-src *; frame-src *; child-src *; frame-src *; worker-src *"
```

Comment:
- `frame-src` is already there.
- `child-src` could be removed.
- `worker-src` should probably be set to `'self'` instead of `*`, or just removed to fallback to `default-src`.

Reply:
Oh, I am blind. Yeah, sounds reasonable, but `child-src` could be useful to be compatible with old browsers. But I guess it's not our case according to app manifest.